RELEASE.md

```diff
@@ -1,40 +1,78 @@
 
-Changes to `Transfer`:
-- As `Asset` is defined as something that can be transferred, the modelling of `Transfer` has been refactored to act upon `Asset` rather than `Observable` with a change to `TransferBase`.
-- This also results in changes to the `Qualify_SecurityTransfer` function.
+Updates to `AssetBase` data type:
+- The `identifier` attribute has been made mandatory.
+- The metadata scheme has been removed.
+- The attributes `isExchangeListed`, `exchange` and `relatedExchange` have been moved from `InstrumentBase` to `AssetBase`.
+- The type of the `exchange` and `relatedExchange` attributes has been updated from `string` to `LegalEntity`.
 
-Product Model:
-- Introduce a new data type on `Payout`: `SettlementCommitment` which models the settlement of an `Asset` for cash.
-- Introduce `TransferableProduct` as a type of Product which can be used in a `SettlementCommitment` for a basic cash settled trade of either an `Asset` with or without the addition of specific `EconomicTerms`.
-- Define the new `SettlementCommitment` data type to model this new kind of `Payout`.
+Updates to `Cash` data type:
+- Two new conditions have been added to ensure that `taxonomy` and `exchange` are not used for this asset type.
+
+Changes to `Commodity` data type:
+- Now extends from `AssetBase` not `ProductBase`.
+- Accordingly, `productTaxonomy` has been replaced by `taxonomy` and the conditions updated.
+
+Changes to `Index` and `IndexBase` data types:
+- The attributes `exchange` and `relatedExchange` have been removed from `IndexBase` as they will now be picked up from 'AssetBase`.
+- `Index` has been refactored from a `one-of` condition to a choice data type.
+- The data type `FloatingIndex` has been renamed `FloatingRateIndex`.
+- The documentation in the model on the index-related data types has been improved.
+- The qualification logic has been updated to reflect the new modelling of `Index` as an underlier on the relevant functions. 
+
+Updates to `ListedDerivative` data type:
+- Updates have been made to the attribute names and types introduced in the previous release to improve modelling and composition.
+- The condition on `VarianceReturnTerms` has been updated to reflect the new position of `ListedDerivative` in the product model.
+
+Changes to `Security` data type:
+- Now extends from `InstrumentBase` not `ProductBase`.
+- Temporary changes made to add `economicTerms` and `productTaxonomy` pending further refactoring in the third phase.
+
+Support for FX Observables:
+- The data type `ForeignExchangeRate` has been created and added as a choice to `Index`; it also extends `IndexBase`.
+- This new data type contains the same attributes as the existing `FXRateObservable` which has been deprecated.
+- The `ExchangeRate` and `ForeignExchange` data types have been deprecated and the deprecated `CrossRate` data type has been deleted.
```

Comment:
`ForeignExchange` has been deprecated. `ExchangeRate` and `CrossRate` have both been deleted.

Reply:
corrected